staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_discovery.go

```diff
@@ -390,6 +393,8 @@ func (dm *discoveryManager) syncAPIService(apiServiceName string) error {
 	}
 
 	dm.mergedDiscoveryHandler.AddGroupVersion(gv.Group, entry)
+	dm.mergedDiscoveryHandler.SetGroupPriority(gv.Group, info.groupPriority)
```

Comment:
Im curious what the behavior is of today's discovery when two APIServices have disagreeing `GroupPriorityMinimum`. Which wins?

Reply:
Looks like it is the group with the higher minimum priority, as the name implies. This PR should account for that